tests/draft3/ref.json

```diff
@@ -141,6 +141,32 @@
         ]
     },
     {
+        "description": "ref, containing other keywords",
+        "schema": {
+            "definitions": {
+                "reffed": { "type": "string" }
+            },
+            "properties": {
+                "foo": {
+                    "$ref": "#/definitions/reffed",
+                    "type": "integer"
```

Comment:
But that's true anyway, isn't it?

```JSON
"properties": {
    "foo": {
        "type": "array",
        "minLength": 10,
    }
}
```
validates `"foo": []` and `"foo": [1, "a", {}]` but not `"foo": "short"` or `"foo": "thisiswaytoolong"`

because `"minLength"` can only fail validation against string instances.  It has no effect on other types.